src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -110,70 +115,75 @@ export default ajaxCaller(class ShareWorkspaceModal extends Component {
         h(FormLabel, { htmlFor: id }, ['User email']),
         h(AutocompleteTextInput, {
           id,
+          openOnFocus: true,
+          placeholderText: _.includes(searchValue, aclEmails) ?
+            'This email has already been added to the list' :
```

Comment:
Really appreciate this check - with the old version I could just keep pasting the same email and adding them over and over and only got an error message at the end when trying to save